internal/vxproto/proto.go

```diff
@@ -854,12 +889,139 @@ func (vxp *vxProto) GetIMCModule(token string) IModuleSocket {
 	return nil
 }
 
+// GetIMCTopic is function which get topic info for imc topic token
+// The function may return nil if the topic token doesn't exist
```

Comment:
```suggestion
// GetIMCTopic returns a topic information object for a IMC topic token
```

I would also suggest to return reference to an object '*tokenInfo' rather then interface, from caller perspective it would be much clearer what is returned and caller then can decide what part of public interface for an object it want to use. (Also I don't see any reason to not make 'TokenInfo' a public type)

Reply:
`GetIMCTopic` is helper public function so:
1. return object type must be public;
2. return object contains only getters;
3. interface is simpler to mock in tests;